### PR TITLE
Updated Kubeflow to latest release (v0.5.0).

### PR DESCRIPTION
**Pull request by michael-balint**

### CODE REVIEW

**Comment by michael-balint** [comment]
@dholt - to test, it's best to increase the VM resources, here was my Vagrantfile...

```
$ cat Vagrantfile-ubuntu 
VAGRANTFILE_API_VERSION = "2"
BOX_IMAGE = "generic/ubuntu1804"

Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
  config.ssh.insert_key = false
  config.ssh.private_key_path = ["~/.ssh/id_rsa", "~/.vagrant.d/insecure_private_key"]
  config.vm.provision "file", source: "~/.ssh/id_rsa.pub", destination: "~/.ssh/authorized_keys"

  config.vm.define "virtual-mgmt" do |mgmt|
    mgmt.vm.provider "libvirt" do |v|
      v.memory = 16384
      v.cpus = 8
    end
    mgmt.vm.box = BOX_IMAGE
    mgmt.vm.network :private_network, ip: "10.0.0.2"
  end

  #config.vm.define "virtual-login" do |login|
  #  login.vm.provider "libvirt" do |v|
  #    v.memory = 2048
  #    v.cpus = 4
  #  end
  #  login.vm.box = BOX_IMAGE
  #  login.vm.network :private_network, ip: "10.0.0.4"
  #end

  config.vm.define "virtual-gpu01" do |gpu|
    config.vm.provider "libvirt" do |v|
      v.memory = 16384
      v.cpus = 8
      v.machine_type = "q35"
      v.cpu_mode = "host-passthrough"
      # comment in for pci passthrough (and change bus according
      # to local hw setup - `lspci -nnk | grep NVIDIA`)
      v.pci :bus => '0x07', :slot => '0x00', :function => '0x0'
      v.pci :bus => '0x08', :slot => '0x00', :function => '0x0'
      v.pci :bus => '0x0e', :slot => '0x00', :function => '0x0'
      v.pci :bus => '0x0f', :slot => '0x00', :function => '0x0'
    end
    gpu.vm.box = BOX_IMAGE
    gpu.vm.network :private_network, ip: "10.0.0.11"
  end

  config.vm.provision "shell", inline: <<-SHELL
	sed -i -e 's/4\.2\.2\.1/8.8.8.8/g' -e 's/4\.2\.2\.2/8.8.4.4/g' /etc/netplan/01-netcfg.yaml
	netplan apply
  SHELL
end
```

File: scripts/k8s_deploy_kubeflow.sh [hunk 1]
```diff
@@ -42,40 +37,29 @@ if [ $? -eq 0 ] ; then
...
     sudo mv ${tempd} ${KUBEFLOW_SRC}
 fi
 
 pushd ${HOME}
-${KUBEFLOW_SRC}/scripts/kfctl.sh init ${KFAPP} --platform none
+${KUBEFLOW_SRC}/kfctl init ${KFAPP}
 cd ${KFAPP}
-${KUBEFLOW_SRC}/scripts/kfctl.sh generate k8s
-pushd ks_app
-ks param set jupyter serviceType NodePort
```
Comment (dholt) [inline scripts/k8s_deploy_kubeflow.sh]: Can we update this to something like `ks param set ambassador ambassadorServiceType LoadBalancer` if the load balancer is deployed?

Does kubeflow default to exposing ambassador on a node port now (vs cluster IP before)?

Comment (michael-balint) [reply scripts/k8s_deploy_kubeflow.sh]: Ah yes, we should totally do that.

And yes, it appears that Kubeflow defaults to exposing Ambassador on NodePort.

Comment (michael-balint) [reply scripts/k8s_deploy_kubeflow.sh]: Done!

**Pull request merged**